src/views/Post.vue

```diff
@@ -1,14 +1,228 @@
+                    >สู้ๆ นะ เดี๋ยววันพรุ่งนี้ก็จะสดใสเอง</small
+                  >
+                </a>
+                <a
+                  href="#"
+                  class="list-group-item list-group-item-action flex-column align-items-start"
+                >
+                  <div class="d-flex w-100 justify-content-between">
+                    <h5 class="mb-1">หมีขั้วโลกเหนือ</h5>
+                    <small class="text-muted">3 days ago</small>
+                  </div>
+
+                  <small class="text-muted"
+                    >สู้ๆ นะ เดี๋ยววันพรุ่งนี้ก็จะสดใสเอง</small
+                  >
+                </a>
+              </div>
+            </div>
+            <div class="col-sm-4 pt-3 test-3">
+              <div>
+                <a
+                  href="#"
+                  class="list-group-item list-group-item-action flex-column align-items-start"
+                >
+                  <div class="d-flex w-100 justify-content-between">
+                    <h5 class="mb-1">หมีขั้วโลกเหนือ</h5>
+                    <small class="text-muted">3 days ago</small>
+                  </div>
+
+                  <small class="text-muted"
+                    >สู้ๆ นะ เดี๋ยววันพรุ่งนี้ก็จะสดใสเอง</small
+                  >
+                </a>
+              </div>
+            </div>
+            <div class="col-sm-4 pt-3 test-3">
+              <div>
+                <pie-chart>Pie Chart</pie-chart>
+                <pie-chart>Pie Chart</pie-chart>
+              </div>
+            </div>
+          </div>
+          <nav aria-label="Page navigation example">
+            <ul class="pagination justify-content-center">
+              <li class="page-item disabled">
+                <a class="page-link" href="#" tabindex="-1">Previous</a>
+              </li>
+              <li class="page-item"><a class="page-link" href="#">1</a></li>
+              <li class="page-item"><a class="page-link" href="#">2</a></li>
+              <li class="page-item"><a class="page-link" href="#">3</a></li>
+              <li class="page-item">
+                <a class="page-link" href="#">Next</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </div>
+  </main>
 </template>
 
-<script>
+<script type="text/javascript">
+// import { PieChart } from "vue-chartjs";
 
+import { Bar } from "vue-chartjs";
+import {
+  Chart as ChartJS,
+  Title,
+  Tooltip,
+  Legend,
+  BarElement,
+  CategoryScale,
+  LinearScale,
+} from "chart.js";
+ChartJS.register(
+  Title,
+  Tooltip,
+  Legend,
+  BarElement,
+  CategoryScale,
+  LinearScale
+);
+export default {
+  components: { Bar },
+  data() {
+    return {
+      chartData: {
+        labels: ["January", "February", "March", "April", "May", "June", "July"],
+        datasets: [{ data: [40, 20, 12] }],
+      },
+      chartOptions: {
+        responsive: true,
+      },
+    };
+  },
+};
 </script>
```

Comment:
actually, I want to convert Composition API  more than Option API, because if you use Composition you should use it anywhere.